### DISCUSSION TITLE
Changing Baudrate

### DISCUSSION THREAD

**Discussion by vunnies** [arendst/Tasmota#15784] [Q&A] [answered]
Hi I'm completely new to Tasmota and I'm trying to connect a P1 smartmeter. I bought This board https://enri.nl/assets/img/pcb/p1-v2-schema.png and it has a Wemos with Tasmota installed.

Sinds I have a Kamstrup 162 meter with DSRM version 2.2 I need t change the baudrate to 9600.. I tried following the manual for changing the baudrate but for some reason it stays at 115200..

What do I have to type to change it

Kind regards Vincent

**Comment by barbudor** [comment, marked as answer]
Did you tried `Baudrate 9600` then `Restart 1`?

**Reply by vunnies** [reply]
I did now.. Restart 1 did not the trick unpluging the Wemos and repower it again did...

Like always     "did you try turning it off and on again ;-) "
